### DISCUSSION TITLE
How to extend default config from another private repo

### DISCUSSION THREAD

**Discussion by fabio-desimone** [renovatebot/renovate#25524] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I would like to extend the default config from another private repo.

### What you tried so far.

I created the `renovate-config` repo and copied a working config from another repo in the `default.json` file.
Then tried to extend the default config in a new repo using the following:
```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "description": "Preset configuration for Renovate",
    "extends": ["github>My-Company/renovate-config"],
    "hostRules": [
        {
            "matchHost": "https://api.github.com/repos/My-Company",
            "encrypted": {
                "token": "<encrypted_ghp>"
            },
            "hostType": "github"
        }
    ]
}
```

Also tried with other host rules:
```json
"hostRules": [
    {
        "hostType": "git-tags",
        "encrypted": {
            "token": "<encrypted_ghp>"
        }
    }
],
```
which is the one I'm currently using in the `renovate.json` files in multiple repos and it works well when updating dependencies from other private repos

also tried:
```json
"hostRules": [
    {
        "hostType": "github",
        "encrypted": {
            "token": "<encrypted_ghp>"
        }
    }
],
```

Thanks in advance for your help

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
you can't extend from a private preset if the required host rule inside same config. the hostrule needs to be in bot/ admin config.

**Reply by fabio-desimone** [reply]
Thank you for your answer.

> the hostrule needs to be in bot/ admin config

I can't find this section. In which file should it go?

**Comment by rarkins** [comment, marked as answer]
If both repos have the bot installed, and both are private, then it should work without further config
